src/mprisplayerinterface.cpp

```diff
@@ -40,10 +41,10 @@ MprisPlayerInterface::MprisPlayerInterface(const QString &service, const QString
     , m_canPause(false)
     , m_canPlay(false)
     , m_canSeek(false)
-    , m_loopStatus(Mpris::LoopNone)
+    , m_loopStatus(QMetaEnum::fromType<Mpris::LoopStatus>().valueToKey(Mpris::LoopNone))
```

Comment:
Maybe, but it seems this wouldn't be exactly a trivial change as the D-Bus interface still must look outside like it has strings. These are stored as part of the interface class and their setter and getter use methods from the parent class.

Reply:
That's not very nice. String constants instead that are already the right value?

Reply:
Do you mean that I should have `m_loopStatus("LoopNone")` instead of using `QMetaEnum`?

I tried a bit if I could replace the underlying implementation with enum values but at least the first try seems to just lead to a crash. I think I should make a task about it and leave it for later as there are more important things to figure out right now.